### ISSUE TITLE
I cannot get any results.

### ISSUE THREAD

**Issue by Kaiyle404** [neojato/T-Hawk#12]
I followed all the instructions on the README but after several hours and no errors I am not seeing anything on Twitter.
I even looked for a different twitter library (on more recent) and still nothing happened

Is this the whole code or is this only a small piece?
What do I need to add to make it work?

**Comment by stritti** [comment]
My forked version works still fine: https://stritti.github.io/bot_sarcoidosis/

Have you followed all the detailed steps in the Readme? Please cross-check this.

**Comment by Kaiyle404** [comment]
> My forked version works still fine: https://stritti.github.io/bot_sarcoidosis/
> 
> Have you followed all the detailed steps in the Readme? Please cross-check this.

I have followed it but I'm pretty much 100% new to code so I'm not sure how to run it really to get a result from it.

**Comment by stritti** [comment]
Maybe you have not already the notification of Twitter to be allowed to use the App with the API key? In my case it took some days.

**Comment by Kaiyle404** [comment]
Oh no I have it. I've had it for two years.

**Comment by stritti** [comment]
Hm. Then I have no more ideas.
Probably you add some logs to the code to get more information.

**Comment by SuperArmySoldiers** [comment]
yeah not running for me either

**Comment by neojato** [comment]
Apologies for the delayed response, it seems the error message from the library we're using isn't fully exposing the issue. See here: https://github.com/airhadoken/twitter-lib/issues/37

tl;dr - You now need [Elevated access for your Project](https://developer.twitter.com/en/portal/products/elevated)
